### PR TITLE
completed css-brochure-form. Paul Son

### PR DESCRIPTION
**Pull request by contactpaulson**
css-brochure-form

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Paul, the width of your form is too wide as compared to the example, please revise and resubmit with a new screenshot

**Comment by contactpaulson** [comment]
css-brochure-form

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -61,5 +61,5 @@ button {
   border: none;
   padding-top: 7px;
   padding-bottom: 7px;
-  margin-left: 1065px;
+  margin-left: 875px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: Hey dude! Please fix this margin, as we discussed 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by contactpaulson** [comment]
css-brochure-form

**Comment by contactpaulson** [comment]
css-brochure-form

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -61,5 +68,10 @@ button {
   border: none;
   padding-top: 7px;
   padding-bottom: 7px;
-  margin-left: 875px;
+}
+
+.newButton {
+  display: flex;
+  justify-content: center;
+  margin-left: 297px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: You can remove all this and just do `text-align: right` as we discussed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by contactpaulson** [comment]
css-brochure-form

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**